app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -111,8 +111,31 @@ object ScenesToProjects extends TableQuery(tag => new ScenesToProjects(tag)) wit
       )
     }
   }
+  
+  def setColorCorrectParamsBatch(projectId: UUID, params: BatchParams)(implicit database: DB) = {
+    val results = for {
```

Comment:
That's a somewhat tangential issue. Sequencing the actions into a single transaction would roll back if there were a DB error, but not if the `run` invocation returned an unexpected number of rows affected. Slick's documentation was a bit vague but I got the impression that rolling back the entire transaction based on the actual result of a single action wasn't possible.